src/app/responders/common/dndefine.py

```diff
@@ -0,0 +1,101 @@
+"""
+Definition webhook responder
+"""
+import logging
+import urllib2
+
+from app.responders.slack import SlackResponder
+from settings import DNDFINE_TOKENS
+
+
+class DndefineResponder(object):
+    """ Define a word! """
+    KEY_WORD = 'dndfine'
+    TOKENS = DNDFINE_TOKENS
+
+    def get_definitions(self, term):
+        logging.info('Searching checking dict for term: %s', term)
+        if term.lower() == "list":
+            return self._format_response(term, '\n'.join(DEFINITIONS.iterkeys())) 
+        return self._format_response(term, DEFINITIONS.get(term.lower()))
+
+    @staticmethod
+    def _format_response(term, definition):
+        response = ''
+
+        response += '*{}*\n\n'.format(term)
```

Comment:
This is just waiting to blow up when some yokel tries a unicode character in their term "for fun"